packages/twenty-front/src/modules/ui/field/display/components/PhonesDisplay.tsx

```diff
@@ -30,6 +38,29 @@ const StyledContainer = styled.div`
 `;
 
 export const PhonesDisplay = ({ value, isFocused }: PhonesDisplayProps) => {
+  const { enqueueSnackBar } = useSnackBar();
+  const { getIcon } = useIcons();
+  const IconClipboard = getIcon('IconClipboardCopy');
+  const handlePhoneClick = async (event: React.MouseEvent, phoneNumber: string | undefined) => {
+    event.preventDefault(); // Prevent default link behavior
+    
+    try {
+      if(phoneNumber !== undefined){
```

Comment:
is isDefined helper + linter does not seem to be effective (npx nx run twenty-front:lint:ci)